### DISCUSSION TITLE
CLI Build

### DISCUSSION THREAD

**Discussion by mgorbach** [MolotovCherry/virtual-display-rs#60] [General]
Would it be possible to get a build with the CLI? I’m looking to script some automatic changes to the virtual monitor based on whether a physical monitor is connected.

**Comment by MolotovCherry** [comment]
Yes, I just need to make some edits to the build system to get it building. I'll get back to this once I finish those edits.

**Comment by MolotovCherry** [comment]
Here you go. You can uninstall the driver completely. Follow the manual driver installation steps [here](https://github.com/MolotovCherry/virtual-display-rs?tab=readme-ov-file#how-to-install-portable-version) (you can skip the registry file and certificate steps)

Please remember, the current master has no working gui tool as of yet, but the cli tool (though not final) works perfectly fine.
~~Iirc there's also no persistence at the moment either~~

But other than that, the driver is pretty much final release ready

_Download removed, please see #115 for latest downloads_
